src/cli/user/login.rs

```diff
@@ -58,7 +66,7 @@ impl Options {
 
         // TODO: graceful timeout, so that users are not forced to Ctrl+C if the server errors
         let secret = res.access_token().secret();
-        keyring::Entry::new("drawbridge", "enarx")
+        keyring::Entry::new("enarx", "drawbridge")
```

Comment:
```suggestion
        keyring::Entry::new("enarx", oidc_domain)
```
Assuming arbitrary strings are allowed here